### ISSUE TITLE
Trying to run test in a project with flavors fails. Main page documentation refers to non-existent version '3.0.0'

### ISSUE THREAD

**Issue by ZenonSeth** [Karumi/Shot#61]
### Expected behaviour
With a project set-up with multiple build flavors, when I correctly specify the `instrumentationTestTask` as such:
```groovy
shot {
    appId = 'com.my.app'
    instrumentationTestTask = 'connectedDevDebugAndroidTest'
  }
```
The test should be able to pass.

Additionally, the documentation on the main page says:
> Remember also you should use Shot 3.0.0 because this feature was introduced in this release!

However the latest released version is 2.2.0 (and as far as I can tell, the next planned is 2.3.0)

### Actual behaviour
When running command
`./gradlew executeScreenshotTests -Precord --stacktrace`
I get the following error:
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:downloadScreenshots'.
> Task with path 'packageDebugAndroidTest' not found in project ':app'.

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.internal.tasks.TaskDependencyResolveException: Could not determine the dependencies of task ':app:downloadScreenshots'.
...
```
(I can provide full stack trace if necessary, but it doesn't seem helpful)

### Steps to reproduce
As far as I can tell, just need a project with multiple build flavors to get this issue.

### Version of the library
2.2.0

**Comment by pedrovgs** [comment]
Hi @ZenonSeth sorry for the error. I had no time to publish the release 3.0.0 but the documentation was updated explaining how to configure your build.gradle file for this version. My bad.

If you update the library to the version 3.0.0, which is being released right now, you should get the library working properly.

Please, let me know if you find any error during the library configuration or the tests execution.

**Comment by ZenonSeth** [comment]
Hi @pedrovgs - thanks for the update, I'll test it as soon as time permits!

**Comment by pedrovgs** [comment]
Hi @ZenonSeth today I've released Shot 3.0.1. I'm closing this issue 😃
